src/app/tech-carbon-estimator/tech-carbon-estimator.component.html

```diff
@@ -9,6 +9,14 @@ <h1 class="tce-text-3xl tce-mb-6">Technology Carbon Estimator</h1>
           aria-live="polite"
           (closeEvent)="closeAssumptionsAndLimitation($event)"></assumptions-and-limitation>
       } @else {
+        <div class="tce-flex tce-justify-end tce-pb-4 -tce-mt-2">
+          <button
+            #showAssumptionsLimitationButton
+            class="tce-button-assumptions tce-px-3 tce-w-fit tce-self-end"
```

Comment:
Was it intentional to drop the `tce-py-2` here? I'm not sure if it had an effect on the Scott Logic styling but just looking at the npm-package styling it's now a bit small:

![image](https://github.com/ScottLogic/sl-tech-carbon-estimator/assets/117279304/f0990fef-3dcd-4fb6-abd7-b99edd3b44e2)

Reply:
If I remember correctly this was intentional and it was because the spacing looked a bit large with the Scott Logic styling. I'll have a look at both versions and see what I can do.

Reply:
Hmm, I wonder if something's caused a priority change, as it looks like the sl-styles.css is already trying to override padding to a small amount for this class.

Reply:
I've added the padding back in. I think it actually looks fine with the Scott Logic styling.

Reply:
Yep, looks good to me too. Could have been something odd with styles updating while still serving locally, doesn't always seem to update fully.